lib/publiccloud/utils.pm

```diff
@@ -176,6 +177,10 @@ sub is_container_host() {
     return is_public_cloud && get_var('FLAVOR') =~ 'CHOST';
 }
 
+sub is_slem_on_pc() {
+    return is_public_cloud && get_var('FLAVOR') =~ 'Micro';
+}
+
```

Comment:
I think we use this only in public cloud scenarios anyways, so maybe `version_utils::is_sle_micro()` would be enough?